src/opm/parser/eclipse/EclipseState/Tables/TableManager.cpp

```diff
@@ -433,7 +434,6 @@ DensityTable make_density_table(const GravityTable& gravity) {
 
         addTables( "OILVISCT", m_tabdims.getNumPVTTables());
         addTables( "WATVISCT", m_tabdims.getNumPVTTables());
-        addTables( "GASVISCT", m_tabdims.getNumPVTTables());
```

Comment:
Is this change accidental or did you mean to remove the `GASVISCT` table declaration?